Jenkinsfile

```diff
@@ -93,40 +93,33 @@ pipeline {
         }
         stage('Full Build') {
             when { expression { env.CHANGE_ID == null } }
-            parallel {
-                stage ('Linux') {
-                    steps {
-                        withMaven(maven: 'Maven 3.5.4', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
-                              sh 'mvn clean install -B $DISABLE_DOWNLOAD_PROGRESS_OPTS'
-                        }
+             options {
+                timeout(time: 30, unit: 'MINUTES')
+            }
+            stage ('Linux') {
```

Comment:
Looks like it is mad, because there is a `stage` embedded within another `stage`: https://jenkins.phx.connexta.com/service/jenkins/blue/organizations/jenkins/codice-maven/detail/PR-15/1/pipeline/

Reply:
I gotta setup my validation again. thanks!